app/bundles/CampaignBundle/Controller/CampaignController.php

```diff
@@ -143,6 +149,140 @@ public function cloneAction(Request $request, $objectId)
         return $this->cloneStandard($request, $objectId);
     }
 
+    /**
+     * Export an entity.
+     *
+     * @return JsonResponse|BinaryFileResponse|Response
+     */
+    public function exportAction(ExportHelper $exportHelper, CampaignModel $campaignModel, int $objectId)
+    {
+        if (!$this->security->isAdmin() && !$this->security->isGranted('campaign:export:enable', 'MATCH_ONE')) {
+            $this->logger->error('Access denied for campaign export', ['user' => $this->user->getId()]);
+
+            return $this->accessDenied();
+        }
+
+        $campaign = $campaignModel->getEntity($objectId);
+
+        if (empty($campaign)) {
+            $this->logger->error('Campaign not found for export', ['objectId' => $objectId]);
+
+            return $this->notFound();
+        }
+
+        $date           = (new \DateTimeImmutable())->format('Y-m-d H:i:s');
```

Comment:
Use constants over strings
```suggestion
        $date           = (new \DateTimeImmutable())->format(DateTimeHelper::FORMAT_DB);
```

Reply:
fixed

Reply:
fixed